.circleci/config.yml

```diff
@@ -22,16 +22,19 @@ references:
     environment:
       CONFIG_ENV: release
       DETECT_CHROMEDRIVER_VERSION: 'false'
-      NODE_ARGS: --max_old_space_size=3072
+      NODE_OPTIONS: --max_old_space_size=3072
```

Comment:
I made this change (amongst others) in #44696 - in addition, I think the usage of default bash environmnet variable is incorrent: `https://github.com/Automattic/wp-calypso/pull/44696/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R78`:

`node ${NODE_ARGS:---max_old_space_size=8192} ` should be `NODE_OPTIONS=${NODE_OPTIONS:---max-old-space-size=8192}`. Node expects either the raw argument `node --max-old-space-size=8192 file.js` OR the enviornment variable NODE_OPTIONS. Settings NODE_OPTIONS with `${NODE_OPTIONS:---max-old-space-size=8192}` (i.e. without left-hand assignment) is incorrect, and is easy to test. A node script run against this invocation will resolve `NODE_OPTIONS` as `undefined`.